src/main/java/com/poiji/config/LogCasting.java

```diff
@@ -0,0 +1,334 @@
+package com.poiji.config;
```

Comment:
we must use a test case to improve our coverage percentage

Reply:
Sure, I totally understand. I wasn't sure how the feature would be received so I didn't completely flesh out the example implementation with proper tests. I'll work on writing these over the next few days as time permits.